addons/html_editor/static/src/others/embedded_components/plugins/excalidraw_plugin/excalidraw_dialog/excalidraw_dialog.js

```diff
@@ -20,7 +20,13 @@ export class ExcalidrawDialog extends Component {
         super.setup();
         this.state = useState({});
         this.inputRef = useAutofocus({ refName: "urlInput" });
+        this.insertButtonRef = useRef("insertButton");
         useExternalListener(window, "keydown", this.onKeyDown.bind(this));
+        onMounted(() => {
+            if (this.insertButtonRef.el) {
+                this.state.isInputEmpty = true;
+            }
+        });
```

Comment:
Why do you need to set it as true in the onMounted and not as a base value in the state variable?

Reply:
If you move it there the ref you added could also be removed as it becomes essentially useless.